rir/src/compiler/opt/delay_instr.cpp

```diff
@@ -17,7 +17,12 @@ void DelayInstr::apply(RirCompiler&, ClosureVersion* function,
             auto next = ip + 1;
 
             if (!i->hasEnv() && !i->hasEffect() && !Phi::Cast(i) &&
-                !i->branchOrExit()) {
+                !i->branchOrExit() &&
+                // TODO: Find out why moving these instructions breaks simple
+                // ranges (seems to mess up the SSA allocator).
+                // For now, these are only used by compiled specials, and never
+                // get optimized by being delayed anyways
+                !Inc::Cast(i) && !Dec::Cast(i) && !SetShared::Cast(i)) {
```

Comment:
is this still the case when you rebase?

Reply:
This was since Saturday, unless any changes were made then

Reply:
Nevermind, looks like this section is completely gone in the latest `master`